src/docs/templates/sweepstakes.hbs

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	{{> html-head title="Sweepstakes" }}
+</head>
+
+<body>
+    {{> sections/header}}
+    <section class="sweepstakes bg-primary-light">
+        <div class="container">
+            <figure class="row m-0 align-items-center">
```

Comment:
FIGURE and FIGCAPTION must be used when the text is strictly related to the image, kind of description and it's short. Please just replace them with a DIV.